### PR TITLE
Show protection symbol if needed on default branch (#7660)

### PR DESCRIPTION
**Pull request by 6543**
missed part of https://github.com/go-gitea/gitea/pull/7660

@gary-kim  here it is :D

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by gary-kim — APPROVED** [review]

File: templates/repo/branch/list.tmpl [hunk 1]
```diff
@@ -13,10 +13,14 @@
 				<tbody>
 					<tr>
 						<td>
-							{{if .IsProtected}}
-								<i class="octicon octicon-shield"></i>
+							{{range $branch := .Branches}}
```
Comment (lafriks) [inline templates/repo/branch/list.tmpl]: Imho if you define loop variable .IsProtected still refers to parent object, no?

Comment (lunny) [reply templates/repo/branch/list.tmpl]: Oh, missed that, you should use `{{range .Branches}}`

**Review by lunny — CHANGES_REQUESTED** [review]

File: templates/repo/branch/list.tmpl (hunk 1 above)
Comment (6543) [reply templates/repo/branch/list.tmpl]: but we shut fix that upstream too!!!

**Review by 6543 — COMMENTED** [review]
https://github.com/go-gitea/gitea/pull/7674/

**Review by lunny — APPROVED** [review]

**Pull request merged**